src/test/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListenerTest.java

```diff
@@ -133,23 +165,24 @@ public void testOnCompletedWithDurationAsZero() throws Exception {
         when(run.getParent()).thenReturn(job);
 
         datadogBuildListener.onCompleted(run, mock(TaskListener.class));
+        Assert.assertEquals(ConcurrentMetricCounters.Counters.get().size(), 1);
 
         String[] expectedTags = new String[4];
         expectedTags[0] = "job:ParentFullName/JobName";
         expectedTags[1] = "node:test-node";
         expectedTags[2] = "result:SUCCESS";
         expectedTags[3] = "branch:test-branch";
         client.assertMetric("jenkins.job.duration", 0, "null", expectedTags);
+        client.assertMetric("jenkins.job.leadtime", 0, "null", expectedTags);
         client.assertServiceCheck("jenkins.job.status", 0, "null", expectedTags);
         client.assertedAllMetricsAndServiceChecks();
     }
 
     @Test
-    public void testOnCompletedSuccessWithStatsD() throws Exception {
+    public void testOnFailures() throws Exception {
```

Comment:
What value does this tests bring compare to the `testOnCompletedWithEverything` one?

Reply:
`testOnCompletedWithEverything` only tests a run that succeeds:  https://github.com/DataDog/jenkins-datadog-plugin/blob/fd3fe2f26a80f7b595f99c04e5e68bd8114496e7/src/test/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListenerTest.java#L93. For `testOnFailures()`, we're testing 2 different metrics (`mtbf` and `feedbacktime`).